crates/core/src/formatter.rs

```diff
@@ -843,20 +842,20 @@ pub fn print_transaction_summary(
         _ => "⚠️",
     };
 
-    tracing::info!("{}  [{}] Hash: {:?}", emoji, status, tx.hash());
-    tracing::info!("Initiator: {:?}", tx.initiator_account());
-    tracing::info!("Payer: {:?}", tx.payer());
-    tracing::info!(
+    sh_println!("\n{}  [{}] Hash: {:?}", emoji, status, tx.hash());
+    sh_println!("Initiator: {:?}", tx.initiator_account());
+    sh_println!("Payer: {:?}", tx.payer());
+    sh_println!(
         "Gas Limit: {} | Used: {} | Refunded: {}",
         to_human_size(tx.gas_limit()),
         to_human_size(used_gas),
         to_human_size(tx_result.refunds.gas_refunded.into())
     );
-    tracing::info!(
+    sh_println!(
         "Paid: {:.10} ETH ({} gas * {})",
         paid_in_eth,
         used_gas,
         format_gwei(l2_gas_price.into())
     );
-    tracing::info!("Refunded: {:.10} ETH", refunded_in_eth);
+    sh_println!("Refunded: {:.10} ETH", refunded_in_eth);
```

Comment:
Same thing here potentially. Note: you can use variable in interpolation directly and named parameters like this: `sh_println!("{argument}", argument = "test")`. See [fmt](https://doc.rust-lang.org/std/fmt/index.html)